packages/rest/src/rest.component.ts

```diff
@@ -41,14 +39,15 @@ import {DefaultSequence} from './sequence';
 export class RestComponent implements Component {
   providers: ProviderMap = {
     [RestBindings.SequenceActions.LOG_ERROR.key]: LogErrorProvider,
-    [RestBindings.SequenceActions.FIND_ROUTE.key]: FindRouteProvider,
```

Comment:
What is the implication of this change on existing code that's using `RestBindings.SequenceActions.FIND_ROUTE`?

(The same comment applies to other binding keys that you are renaming in this PR.)